lib/pages/forms_pages/components/whatsapp_camera.dart/camera_whatsapp.dart

```diff
@@ -139,7 +141,7 @@ class WhatsappCamera extends StatefulWidget {
   ///
   ///```
   ///
-  const WhatsappCamera({key, this.multiple = true});
+  const WhatsappCamera({key, this.multiple = true, required this.isAdultReport});
```

Comment:
Wouldn't it be simpler if instead of adding a boolean using a string instead? 

something like `this.InfobadgeText` and only show is that variable is not null or empty? i think it's more scalable

Reply:
So then, the text `MyLocalizations.of(context, 'one_mosquito_reminder_badge')` will be placed in adult_report_page.dart, which makes more sense